src/docker-osx-dev

```diff
@@ -956,12 +958,19 @@ function configure_includes {
 # Runs the docker-osx-dev script to kick off file watching and syncing.
 #
 function watch_and_sync {
+  sync_only
+  watch
+}
+
+#
+# Runs the docker-osx-dev script to syncing one time.
+#
+function sync_only {
```

Comment:
Given that we have functions called `watch` and `watch_and_sync`, it seems like `sync` would be a better name here.